common/app/routes/Challenges/views/classic/Editor.jsx

```diff
@@ -120,11 +124,18 @@ export class Editor extends PureComponent {
     MouseTrap.unbind('e', 'keyup');
   }
 
+  componentDidUpdate(nextProps) {
+    if (nextProps.id !== this.props.id) {
+      this.refs.editor.getCodeMirror().doc.clearHistory();
```

Comment:
Is this still required? 

My understanding is that if `react` is tracking a keyed component and the key changes, it is essentially like the previous keyed component is unmounted and and the new keyed component is mounted, therefore there would be no history to clear.

Reply:
Yes nice catch because of key there is no need to clear history. thanks for input, i will updated the PR with latest.